utils/unminify.js

```diff
@@ -16,35 +16,38 @@
  * @fileoverview
  * Convert's stacktrace line, column number and file references from minified
  * to unminified. Caches requests for and source maps and once obtained.
+ * @type location{
+ * string: lineNumber
+ * string: columnNumber
+ * string: sourceMapUrl
+ * string: sourceUrl
```

Comment:
It contains capture groups to prevent rerunning the regex again. I did not pass the match object directly for readability purposes.